File: src/kyu_8/find_the_first_non_consecutive_number/java/FirstNonConsecutive.java
package kyu_8.find_the_first_non_consecutive_number.java;

import java.util.Arrays;

public class FirstNonConsecutive {
    static Integer find(final int[] array) {
        for (int i = 1; i < array.length; i++) {
            int a = array[i] - array[i-1];
            if (a >= 2) {
                return array[i];
            }
        }
        return null;
    }
}

File: src/kyu_8/find_the_first_non_consecutive_number/js/firstNonConsecutive.js
function firstNonConsecutive(arr) {
    for (let i = 0; i < arr.length; i++) {
        let a = arr[i] - arr[i - 1];
        if (a >= 2) {
            return arr[i];
        }
    }
    return null;
}

module.exports = firstNonConsecutive;